internal/services/springcloud/spring_cloud_gateway_resource.go

```diff
@@ -240,6 +240,7 @@ func resourceSpringCloudGateway() *pluginsdk.Resource {
 			"sso": {
 				Type:     pluginsdk.TypeList,
 				Optional: true,
+				Computed: true,
```

Comment:
this shouldn't be Computed, since this makes it impossible to unset this field - instead the Create and Update will need splitting so that users can use `ignore_changes` as appropriate